### PR TITLE
blog post for Fuse Tooling Mars release on Oct 6

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

File: blog/2016-10-06-fusetooling-release-mars.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+= JBoss Fuse Tooling released for Eclipse Mars 
+:page-layout: blog
+:page-author: lhein
+:page-tags: [fuse, camel, integration, jbosstools, devstudio, jbosscentral]
+
+
```
Comment (apupier) [inline blog/2016-10-06-fusetooling-release-mars.adoc]: somewhere here, we need to put a link to the same article which is planned to be published on developers.red.com

Comment (lhein) [inline blog/2016-10-06-fusetooling-release-mars.adoc]: In your blog posts you simply put this in:

Note: This article has been posted also on http://developers.redhat.com/blog/2016/05/17/jboss-fuse-tooling-diagram-reworked-new-shiny-colors-and-more/[Red Hat Developers Program blog]

Comment (lhein) [inline blog/2016-10-06-fusetooling-release-mars.adoc]: that seems to have no real effect than maybe tracking

Comment (apupier) [inline blog/2016-10-06-fusetooling-release-mars.adoc]: I think that it has some trackback for Google search engines

Comment (lhein) [inline blog/2016-10-06-fusetooling-release-mars.adoc]: added the link

**Pull request merged**